client/components/CandidateTests/CandidateTestPlanRun/index.jsx

```diff
@@ -0,0 +1,698 @@
+                    {`${reviewStatusText} `}
+                </div>
+            </div>
+            <div className="test-info-entity target-date">
+                <div className="info-label">
+                    <b>Target Completion Date: </b>
+                    {targetCompletionDate}
+                </div>
+            </div>
+        </div>
+    );
+
+    const feedback = testPlanReport.issues.filter(
+        issue => issue.testNumberFilteredByAt == currentTest.seq
+    ).length > 0 && (
+        <div className="issues-container">
+            <h2
+                style={{
+                    width: isLaptopOrLarger
+                        ? issuesHeadingSize?.width
+                        : issuesListSize?.width,
+                    position: 'relative'
+                }}
+            >
+                <span className="feedback-from-text">Feedback from</span>{' '}
+                <b>{at} Representative</b>
+            </h2>
+            <ul
+                className="feedback-list"
+                style={{
+                    width: issuesListSize?.width,
+                    position: 'relative'
+                }}
+            >
+                {[changesRequestedIssues, feedbackIssues].map((list, index) => {
+                    if (list.length > 0) {
+                        const uniqueAuthors = [
+                            ...new Set(list.map(issue => issue.author))
+                        ];
+                        const differentAuthors = !(
+                            uniqueAuthors.length === 1 &&
+                            uniqueAuthors[0] === data.me.username
+                        );
+                        return (
+                            <FeedbackListItem
+                                key={`${index}-issues`}
+                                differentAuthors={differentAuthors}
+                                type={
+                                    index === 0
+                                        ? 'changes-requested'
+                                        : 'feedback'
+                                }
+                                issues={list}
+                                individualTest={true}
+                                githubUrl={generateGithubUrl(
+                                    true,
+                                    index === 0
+                                        ? 'changes-requested'
+                                        : 'feedback',
+                                    null,
+                                    true,
+                                    !differentAuthors ? data.me.username : null
+                                )}
+                            />
+                        );
+                    }
+                })}
+            </ul>
+        </div>
+    );
+
+    const results = (
+        <>
+            <h1 className="current-test-title">{currentTest.title}</h1>
+            <DisclosureComponent
```

Comment:
Use of DisclosureComponent

Reply:
Nice! Maybe it would be good to pass a bg-color prop as well so the parent component can control the background color. Based on the design, this background shouldn't be the default color on this component.